app/uk/gov/hmrc/customs/declarations/stub/controllers/UpscanStubController.scala

```diff
@@ -64,33 +63,11 @@ class UpscanStubController @Inject()(appConfig: AppConfig, httpClient: HttpClien
       FileUpload(i.toString, waiting(i.toString), id = s"$i")
     }.toList)
 
-    Ok(XmlHelper.toXml(resp)).as(ContentTypes.XML)
-  }
-
-  def handleS3FileUploadRequest = Action(parse.multipartFormData) { implicit req =>
-    val redirectLocation = req.body.dataParts("success_action_redirect").head
-    val reference = redirectLocation.split("/").last
-    callBack(reference)
-    SeeOther(redirectLocation).withHeaders("Location" -> redirectLocation)
-  }
-
-  def callBack(ref: String)(implicit hc: HeaderCarrier) = {
-    Thread.sleep(1000)
-
-    val notification =
-      <Root>
-        <FileReference>{ref}</FileReference>
-        <BatchId>5e634e09-77f6-4ff1-b92a-8a9676c715c4</BatchId>
-        <FileName>File_{ref}.pdf</FileName>
-        <Outcome>SUCCESS</Outcome>
-        <Details>[detail block]</Details>
-      </Root>
-
-    val url = appConfig.cdsFileUploadBaseUrl + "/internal/notification"
```

Comment:
Can you explain why these are removed?

Reply:
This was the fake notification XML send upon the receipt of an uploaded file. As the stub is no longer being send these files (they are now going to the testOnly endpoint on the SFUS FE) it does not need to send these fake XML notifications.